openmdao/core/indepvarcomp.py

```diff
@@ -139,10 +139,10 @@ def add_output(self, name, val=1.0, shape=None, units=None, res_units=None, desc
             Default is None.
         ref : float
             Scaling parameter. The value in the user-defined units of this output variable when
-            the scaled value is 1. Default is 1.
```

Comment:
What I meant was these lines here. We don't need to describe in detail the default value and the definition of the ref/ref0/etc arguments when we don't even use them.  Just say "This argument is deprecated.  Default can be None. 

So, this:

    ref : float
        Scaling parameter. The value in the user-defined units of this output variable when
        the scaled value is 1. Default is 1.

Should become

    ref : None
        This argument is deprecated.

Reply:
Fix